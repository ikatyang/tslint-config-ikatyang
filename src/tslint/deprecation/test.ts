/** @deprecated */
declare function deprecated(): void;
declare function nonDeprecated(): void;

// good

nonDeprecated();

// bad

deprecated();
