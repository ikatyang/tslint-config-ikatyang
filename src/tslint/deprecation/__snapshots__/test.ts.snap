// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
/** @deprecated */
declare function deprecated(): void;
declare function nonDeprecated(): void;

// good

nonDeprecated();

// bad

deprecated();
~~~~~~~~~~    [deprecated is deprecated.]

`;
